es-extension-examples/ext-example-lit-element/vite.config.js

```diff
@@ -0,0 +1,48 @@
+import reactRefresh from '@vitejs/plugin-react';
+import babel from 'vite-plugin-babel';
+import { defineConfig } from 'vite';
+import dotenv from 'dotenv';
+dotenv.config();
+
+const urlPath = process.env.URLPATH || ''
+
+export default defineConfig({
+  /* when in lib mode, must define process.env */
+  define: {
+    'process.env': JSON.stringify({
+      NODE_ENV: process.env.NODE_ENV,
+    })
+  },
+  base: urlPath,
+  root: '.',
+  plugins: [
+    reactRefresh(),
+    babel(),
+  ],
+  server: {
+    port: 9094,
+    strictPort: true,
+    host: '0.0.0.0',
+    base: '/',
+  },
+  build: {
+    base: urlPath,
+    outDir: "dist",
+    /* lib config necessary to prevent "Failed to load extension Error" */
+    lib: {
+      name: "ext-example-lit-element",
+      entry: ['./src/ext-lit-element.ts', './src/index.ts'],
```

Comment:
all extensions need to be added to lib.entry in order to be correctly transpiled by vite